app/components/CardList.tsx

```diff
@@ -15,6 +16,7 @@ export const CardList = (): JSX.Element => {
         <PoapCard />
         <EnsCard />
         <PohCard />
+        <GithubCard />
```

Comment:
Minor change: This should be moved before BrightidCard so the web2 identity providers are grouped together.

Reply:
Done